### ISSUE TITLE
Sourcing productEditionId

### ISSUE THREAD

**Issue by memphisraynz** [pbatard/Fido#16]
HI,
Great script, fantastic work.
I am trying to modify for use as an unatended script. I am curious on where you are initially sourcing the productEditionId from?
From what I can see this is the key information the script is based off, the Id's dont seem to match up with any other reference I found.

Thanks

**Comment by pbatard** [comment]
1. Wait for Microsoft to officially offer the ISO from https://www.microsoft.com/en-us/software-download/windows10ISO (after changing your `User-Agent` id so that it doesn't redirect you to the MCT download).
2. Go through the ISO download and inspect the HTML queries to find the `productEditionId` for the release you are after.
3. Fiddle with those (+1, +2, -1, -2) to try to "guess" the Chinese and Edu edition.

In short, a close inspection of the official download page HTML will give you the info you want.

Now, for the older ISOs, I think I got some of the Ids from https://tb.rg-adguard.net and other sources. And yes, it was a pain in the ass to collect, and I sure with __other__ people were doing that work and publishing their results, like I do, so that I don't have to.

Hope that answers your question.

**Comment by xbjfk** [comment]
@pbatard Hello, I found this, I think it might be of interest:
https://techbench.luzea.ovh/dump.json

**Comment by pbatard** [comment]
@xbjkf, Nice find.
Looks like dump.json is generated with [this script](https://github.com/luzeagithub/techbench-json-dump/blob/master/main.py), which might come handy.

**Comment by pbatard** [comment]
Just leaving this for reference, knowing that Windows 10 21H1 id is `2033` and that upcoming releases should be in the `2040`-`2200` range.

```
./techbench_dump.py 2000 2200 --json-indent 4 dump.json
```
